packages/webapp/src/_app/hooks/queries.ts

```diff
@@ -305,34 +325,30 @@ export const useCurrentElection = (queryOptions: any = {}) =>
     });
 
 /**
- * use `votes` table data (for a particular person and round)
- * ASSUMPTION: this use method will only be called by *non*-Chief ongoing rounds
- * It relies on election state being active (not final)
+ * Use `votes` table data (for a particular person and round).
+ * This is only enabled during Active and Final ongoing rounds.
  * @param {string} memberAccount - account of member to get
  * @param {number} roundIndex - election round index
+ * @param {any} queryOptions - any react-query queryOptions overrides (enabled merged)
  */
 export const useMemberGroupParticipants = (
     memberAccount?: string,
     roundIndex?: number,
     queryOptions: any = {}
 ) => {
     const { data: currentElection } = useCurrentElection();
-    const currentActiveElection = currentElection as CurrentElection_activeState;
-
-    let enabled = Boolean(memberAccount) && roundIndex !== undefined;
 
-    if ("enabled" in queryOptions) {
-        enabled = enabled && queryOptions.enabled;
+    let config;
+    if (currentElection && "config" in currentElection) {
+        config = currentElection.config;
     }
 
+    let enabled = Boolean(memberAccount) && roundIndex !== undefined;
+
     return useQuery<VoteData[], Error>({
-        ...queryMemberGroupParticipants(
-            memberAccount,
-            roundIndex,
-            currentActiveElection?.config
-        ),
+        ...queryMemberGroupParticipants(memberAccount, roundIndex, config),
```

Comment:
config still ends up being undefined if we're not in the Active state, no? gotta be a more elegant way to do this. Don't know what it is, but there's gotta be. :) no need for now, but worth noodling on. I *think* why we're doing this here is that you can say currentElection?.config due to TS. Otherwise, `currentElection?.config` would be equivalent to what's above.

Reply:
Yes, still `undefined` if we're in the `Active` state. But I used a type guard instead of a cast, since what comes in isn't actually always a `CurrentElection_active`.

I modified the wording the JSDoc here because, technically, the `queryFn` still returns critical data when we pass `undefined` in as `config`.